GoMeet/src/main/webapp/js/meeting-display.js

```diff
@@ -0,0 +1,25 @@
+function toggleMeetingDisplay() {
+  LoginStatus.doGet().then((loginStatus) => {
+    let meetingName = document.getElementById('meeting-title');
+    let meetingTimes = document.getElementById('vote-meeting-times'); 
+    let meetingLocations = document.getElementById('vote-meeting-locations'); 
+    let votesTable = document.getElementById('votes-table'); 
+    
+    if (loginStatus.loggedIn === 'true') {
+      meetingName.style.display = 'block';
+      meetingTimes.style.display = 'block';
+      meetingLocations.style.display = 'block';
+      votesTable.style.display = 'block';
+      let logoutPrompt = '<a href=' + loginStatus.logoutUrl + '>Logout</a>';
+      document.getElementById('login-or-logout-prompt').innerHTML = logoutPrompt;
+    } else {
+      meetingName.style.display = 'none';
+      meetingTimes.style.display = 'none';
+      meetingLocations.style.display = 'none';
+      votesTable.style.display = 'none';
+      let loginPrompt = '<p>Please log in to view the meeting event details</p>'
+      + '<a href=' + loginStatus.loginUrl + '>Login</a>';
+      document.getElementById('login-or-logout-prompt').innerHTML = loginPrompt;
+    }
```

Comment:
Nits: I think we can make this a bit cleaner by doing something like this
`let style = '';  
let prompt = '';
if (loginStatus.loggedIn === 'true') {
  style = 'block';
  prompt = ......  
} else { 
  style = 'none';
  prompt = ....;  
}. 
`

Reply:
Done for the 'prompt' variable. However, if I try to do this to the style lines it seems to just save the value of XXX.style.display in a variable. So when I try to update the value to 'block', it updates the variable value not the actual XXX.style.display value.

Reply:
Can you try `xxx.style.display = style.valueOf()`?

Reply:
<img width="640" alt="Screen Shot 2021-02-01 at 12 00 35 pm" src="https://user-images.githubusercontent.com/75598641/106403980-2a6ac380-6485-11eb-962a-aac6670f3450.png">
is this what you mean? (lines 8 and 12) if so, it doesn't work either :(

Reply:
I meant something like this
```js
function toggleMeetingDisplay() {
  LoginStatus.doGet().then((loginStatus) => {
    let style = '';
    let prompt = '';

    if (loginStatus.loggedIn === 'true') {
      style = 'block';
      prompt = '<a href=' + loginStatus.logoutUrl + '>Logout</a>';
    } else {
      style = 'none';
      prompt = '<p>Please log in to view the meeting event details</p>'
      + '<a href=' + loginStatus.loginUrl + '>Login</a>';
    }
    
    document.getElementById('meeting-title').style.display = style.valueOf();
    document.getElementById('vote-meeting-times').style.display = style.valueOf();
    document.getElementById('vote-meeting-locations').style.display = style.valueOf();
    document.getElementById('votes-table').style.display = style.valueOf();
    document.getElementById('login-or-logout-prompt').innerHTML = prompt.valueOf();
    
   ```

Reply:
Sorry I'm not too sure what you mean by this. Is it: 
let styleVar = 'block'; 
if (loggedIn === true) {
  xxx.style.display = styleVar.valueOf();

Reply:
done thank you :)

Reply:
LGTM, but why can't we move line 8 and 13 outside of if-else clause?

Reply:
Sorry my bad - moved!

Reply:
Do I need to use .valueOf() ? I think it should still work without

Reply:
I don't think .valueOf() is needed here. The reason why we need .valueOf() of style.display is because it requires to have a value assigned to it (read more here: https://www.w3schools.com/jsref/prop_style_display.asp).

Reply:
@austinct do you know the reason here?

Reply:
@austinct do you know the reason here?

Reply:
Err I'm not too sure where `valueOf()` comes from in this context - `style`/`prompt` are already strings so calling `valueOf()` on it returns itself. Seems like it's not necessary?

Reply:
Ok I've removed it and it still passes the tests :)